umap/static/umap/js/umap.core.js

```diff
@@ -426,6 +426,7 @@ L.U.Keys = {
   I: 73,
   L: 76,
   M: 77,
+  O: 79,
```

Comment:
We could use the [standard mapping](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) instead of using our custom one.

Reply:
Good point! Let's do that in a separate PR ?